### PR TITLE
[WIP] Remove `--save-dev` from component install instructions

### PR DESCRIPTION
**Pull request by AmanDeep9925**
Solving the issue #718 
components

Removing the --save-dev

### CODE REVIEW

**Comment by AmanDeep9925** [comment]
> Solving the issue #718
> components
> 
> Removing the --save-dev

**Comment by adamzerella** [comment]
@AmanDeep9925 Thanks for submitting this PR! 

Before we can merge it though, you'll need to address the spacing and other unrelated formatting changes. 

Making this kind of change is also going to require that the relevant packages are version bumped up. This will involve updating the patch version in the `package.json`, and updating the `CHANGELOG.md` and `README.md` version history.

**Comment by AmanDeep9925** [comment]
@adamzerella I have the repo as you mentioned above, you can review it now.

File: packages/accordion/CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 ## Versions
 
+* [v7.0.8 - Removed developer dependecies of the packages](#v708)
```
Comment (sukhrajghuman) [inline packages/accordion/CHANGELOG.md]: this implies that developer dependencies were removed for each of the packages. 

Try something like

"Remove `--save-dev` flag from read me instructions"

File: packages/accordion/README.md [hunk 2]
```diff
@@ -297,4 +298,4 @@ Licensed under [MIT](https://raw.githubusercontent.com/govau/design-system-compo
 
 **[⬆ back to top](#contents)**
 
-# };
+# };
```
Comment (adamzerella) [inline packages/accordion/README.md]: Whatever the difference in spacing is here we should leave it out of this PR

File: packages/animate/README.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+---
+
+## Install
+
+```shell
+yarn add @gov.au/animate
+```
+
+```shell
+npm install @gov.au/animate
```
Comment (adamzerella) [inline packages/animate/README.md]: The only difference in this file should be the release history and install instruction? We seem to have some spacing changes that make this PR harder to read

**Comment by adamzerella** [comment]
@AmanDeep9925 Thanks again for contributing this, we've moved it over to #748 and will update the components soon :)

**Pull request closed without merging**